src/darwin/Framework/CHIP/templates/MTRCommandPayloadsObjc-src.zapt

```diff
@@ -46,7 +46,13 @@ NS_ASSUME_NONNULL_BEGIN
   auto other = [[MTR{{cluster}}Cluster{{command}}Params alloc] init];
 
   {{#zcl_command_arguments}}
+  {{#if (isProvisional ../cluster command=../command commandField=(asStructPropertyName label))}}
```

Comment:
This is not correct, as @ksperling-apple pointed out.  The framework guts need to handle this no matter what the API is doing.